src/FormFeedback/Tooltip/helpers/__tests__/resolveToltipArrowPosition.test.js

```diff
@@ -0,0 +1,107 @@
+// @flow
```

Comment:
Typo in the file name, should be `resolveTooltipArrowPosition.test.js`, with double "o". I recommend fixing this, otherwise it will be harder to navigate to it with fuzzy search.